homeassistant/components/bluetooth_adapters/const.py

```diff
@@ -0,0 +1,3 @@
+"""Constants for the Bluetooth Adapters integration."""
+
```

Comment:
Let’s merge this into the init file

Reply:
d53d2bbcccec330df6209b870c47c8f5dcef492c